pyproject.toml

```diff
@@ -81,7 +81,7 @@ mkdocstrings-python = "0.8.3"
 requests-mock = "^1.10.0"
 parameterized = "^0.8.1"
 myst-parser = "^0.15.2"
-nautobot-chatops = "^2.0.0"
+nautobot-chatops = { version = "^2.0.2", extras = ["ipfabric"] }
```

Comment:
2.0.2 doesn't exist yet. 2.0.1 was just released. Please update the lockfile after this change.

Reply:
Fixed